sources/packages/web/src/views/student/RestrictionBanner.vue

```diff
@@ -1,14 +1,59 @@
 <template>
-  <Message severity="error">
-    {{ restrictionMessage }}
-  </Message>
+  <banner
+    class="mb-2"
+    v-if="restriction"
+    :type="BannerTypes.Error"
+    header="Your account has a restriction"
+    summary="You have a restriction on your account that must be resolved to continue with your student financial aid. Please view the message and resolve the items to minimize disruption and impact."
+  >
+    <template #actions>
+      <v-btn :color="BannerTypes.Error" @click="viewStudentAccountActivity">
```

Comment:
`<v-btn :color="BannerTypes.Error"` this does not seem right? Either we need a more generic enum or we could use just the strings for this case, which seem to be the Vue way of doing things.